docs/release-notes/latest-changes.md

```diff
@@ -28,6 +28,56 @@ All Rights Reserved. (Contributor contact(s):________________[Insert hyperlink/a
 
 Date       | Change
 ---------  | -------------
+2022-02-03 | fix terminator history
+2022-02-04 | barman: update to 2.18
```

Comment:
fix terminator history was necessary to obsolete the python-3.5 version of terminator. This entry should be removed.

Reply:
Done